### PR TITLE
gpu instructions added

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+Learn more at:
+[➜ golem-gpu-live @ GitHub](https://github.com/golemfactory/golem-gpu-live)
+
+## 1. Download and unzip image
+
+In the first step, download and unzip the `gpu-golem-live` provider image.
+
+To do this, click on the link below or copy the address and paste it in your web browser. The download will begin.
+
+[Download image](https://mirror.notset.fr/golem/golem-gpu-live-231215T091610Z.img.xz)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:35]: The download doesn't begin for me. Instead I just see the content of the file.

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+```bash
+https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
+```
+
+Alternatively, you can download the file using a terminal by typing the command:
+
+```bash
+wget https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:44]: The download mirror is pretty slow. I only get 20 MB/s download speeds where as my connection should do up to 125 MB/s

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+```
+
+The file takes about 2 GB and should download in about 3 minutes (on a 100 Mbps connection).
+
+Once the download is complete, unzip the file with any tool (such as the default archiving tool, The Unarchiver or 7-Zip) or use the command prompt:
+
+```
+gunzip golem-gpu-live-latest.img.xz
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:52]: Nothing unzipped at my end
```
➜  ~ gunzip golem-gpu-live-latest.img.xz
gzip: golem-gpu-live-latest.img.xz: unknown suffix -- ignored
```

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+```
+
+Done! The unzipped file should take up about 8.6 GB.
+
+## 2. Record image to SSD
+
+In the second step, you need to burn the unzipped `gpu-golem-live` image to an external SSD. To do this, you can use free software (such as balenaEtcher) or a command in the terminal.
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:59]: "burn" feels very old school, and might make the providers as seen on the discord channel worry that this is outdated tech. Use the term write or flash

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+
+3.  **Record the image!** - Click on the `Flash!` button to start the recording process
+
+    ![Flash!](/gpu/balena_4.png)
+
+    ⚠️ For larger capacity SSDs, you may see additional confirmation - click `Yes, I'm sure`.
+
+    ![Yes, I'm sure](/gpu/balena_5.png)
+
+4.  **Recording** - in the last step you need to be patient. Recording the video, depending on the speed of the SSD, will take from 5 to 10 minutes. While recording, you will be able to watch the progression in real time.
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:87]: Recording sounds super wrong, and I don't think ive ever seen it used before for this purpose. The correct term would be flashing

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 1](/gpu/balena_6.png)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:89]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 2](/gpu/balena_7.png)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:91]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 3](/gpu/balena_8.png)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:93]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 4](/gpu/balena_9.png)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:95]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 5](/gpu/balena_10.png)
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:97]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 11]
```diff
@@ -0,0 +1,288 @@
...
+
+Once the download is complete, unzip the file with any tool (such as the default archiving tool, The Unarchiver or 7-Zip) or use the command prompt:
+
+```
+gunzip golem-gpu-live-latest.img.xz
+```
+
+Done! The unzipped file should take up about 8.6 GB.
+
+## 2. Record image to SSD
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:57]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 12]
```diff
@@ -0,0 +1,288 @@
...
+1.  **Select a file from disk** - click on the `Flash from file` button to select a previously unzipped `gpu-golem-live` image file.
+
+    ![Flash from file](/gpu/balena_1.png)
+
+2.  **Select SSD** - click on the `Select target` button to select the SSD, which is the place where the `gpu-golem-live` image will be installed.
+
+    ![Select target 1](/gpu/balena_2.png)
+    ![Select target 2](/gpu/balena_3.png)
+
+3.  **Record the image!** - Click on the `Flash!` button to start the recording process
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:79]: Recording

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 13]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Recording 2](/gpu/balena_7.png)
+
+    ![Recording 3](/gpu/balena_8.png)
+
+    ![Recording 4](/gpu/balena_9.png)
+
+    ![Recording 5](/gpu/balena_10.png)
+
+Done! Go to the next step.
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:99]: Remove. Not necessary

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 14]
```diff
@@ -0,0 +1,288 @@
...
+
+    ![Setup GPU Golem Live 6](/gpu/setup_6.png)
+
+8.  **Set your prices** - in the following steps, set the hourly rate (in GLM) and the initial price (in GLM). These values are necessary to participate in the Golem Network and set prices for sharing resources.
+
+    _GLM per hour means how much GLM you will receive for a full hour of calculations._
+
+    ![Setup GPU Golem Live 7](/gpu/setup_7.png)
+
+    _GLM init price means how much GLM you will receive for taking the order. It is recommended to set this value to 0, due to billing for rental time._
```
Comment (cryptobench) [inline src/pages/docs/providers/gpu/gpu-golem-live.md:172]: Init price? Why do we need a new term for start price which is already used on the network?

**Review by cryptobench — CHANGES_REQUESTED** [review]

File: src/pages/docs/providers/gpu/gpu-golem-live.md (hunk 4 above)
Comment (FOMFNY) [reply src/pages/docs/providers/gpu/gpu-golem-live.md:59]: you're right, "flash" will be ok

File: src/pages/docs/providers/gpu/gpu-golem-live.md (hunk 5 above)
Comment (FOMFNY) [reply src/pages/docs/providers/gpu/gpu-golem-live.md:87]: ok -> Flashing

File: src/pages/docs/providers/gpu/gpu-golem-live.md [hunk 15]
```diff
@@ -0,0 +1,288 @@
...
+
+_For example_:
+
+![Golem Node Name](/gpu/nodename.png)
+
+Find your provider online:
+
+![Golem Network Stats 1](/gpu/monitoring_1.png)
+
+And monitor your salary:
```
[the remark this answers is not in the document]
Comment (FOMFNY) [reply src/pages/docs/providers/gpu/gpu-golem-live.md:211]: ok

File: src/pages/docs/providers/gpu/gpu-golem-live.md (hunk 14 above)
Comment (jalas167) [reply src/pages/docs/providers/gpu/gpu-golem-live.md:172]: will not init, as on the screen is init., will keep order for a moment, tbc with

File: src/pages/docs/providers/gpu/gpu-golem-live.md (hunk 3 above)
Comment (jalas167) [reply src/pages/docs/providers/gpu/gpu-golem-live.md:52]: I could unzip without issues on windows

**Review by cryptobench — APPROVED** [review]

**Pull request merged**